float64 current_latitude
float64 current_longitude
bool reached_waypoint